test/ozaria/site/components/interactive/PageInteractive.spec.js

```diff
@@ -2,82 +2,154 @@
 import { mount } from '@vue/test-utils'
 import pageInteractive from 'ozaria/site/components/interactive/PageInteractive/index'
 import factories from 'test/app/factories'
-import * as interactiveApi from 'ozaria/site/api/interactive'
 import draggableOrderingComponent from 'ozaria/site/components/interactive/PageInteractive/draggableOrdering/index'
-import inserCodeComponent from 'ozaria/site/components/interactive/PageInteractive/insertCode'
+import insertCodeComponent from 'ozaria/site/components/interactive/PageInteractive/insertCode'
 import draggableStatementCompletionComponent from 'ozaria/site/components/interactive/PageInteractive/draggableStatementCompletion/index'
 
-const createComponent = (values = {}) => {
-  return mount(pageInteractive, {
-    propsData: values
-  })
-}
-
 const interactive = {
   interactiveType: 'draggable-ordering',
   promptText: 'Interactive prompt text',
-  _id: 'interactiveId1'
+  _id: 'interactiveId1',
+
+  // Have all data options available chosen by type.
+  draggableStatementCompletionData: {
+    elements: [
+      {
+        textStyleCode: true,
+        text: 'Something',
+        elementId: '5d12b22025196e3473b88b8e'
+      }
+    ],
+    labels: [
+      {
+        text: 'label 1'
+      }
+    ]
+  },
+  insertCodeData: {
+    starterCode: 'some code\nline2\nline3\nline4',
+    lineToReplace: 2
+  },
+  draggableOrderingData: {
+    elements: [
+      {
+        text: 'Something',
+        elementId: '5d12b22025196e3473b88b8e'
+      }
+    ],
+    labels: [
+      {
+        text: 'label 1'
+      }
+    ]
+  }
 }
 
-const introLevel = {
-  _id: 'intro-level-id'
-} // Dummy intro level object
+const store = new Vuex.Store({
```

Comment:
Alternately, you can also use the global store and just spy on the apis that the store is using. That way, it will cover testing for the store interactives module as well, and we wont have to update the store here locally in case we update the global interactives store.

Reply:
Agreed - I haven't had a chance to test deeply / look through these.  Added a TODO and will revisit as part of interactive tests after UX.

Reply:
Agree as well. I spent a bit of time trying to implement the api mocking but got bugs.